packages/KTable/KTable.vue

```diff
@@ -148,6 +149,12 @@ export default {
       type: Boolean,
       default: false
     },
+    /**
+     */
+    isClickable: {
```

Comment:
I know I added this but if we’re adding prop descriptions can you add one here? “Adds hover and non selectable styling “ etc